### DISCUSSION TITLE
我不能添加机器人，插件配置页面没有nonebot的配置项

### DISCUSSION THREAD

**Discussion by k4m7v2pz** [koishijs/koishi#760] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/97567575/179883789-8b17aaa7-2e5d-4cdc-961d-924d76d052a8.png)
![image](https://user-images.githubusercontent.com/97567575/179883822-2b0c4f6d-9701-413b-9075-130ab0050183.png)
![image](https://user-images.githubusercontent.com/97567575/179883911-427199bf-b307-4a4d-9f73-a410ed6a9259.png)

**Comment by shigma** [comment, marked as answer]
您是否想问 onebot 而不是 nonebot.jpg

1. 前往「插件配置」
2. 添加插件（点击左侧栏中带有加号的插头按钮）
3. 在右侧选择 @koishijs/plugin-adapter-onebot（如果不好找可以直接在选择框中打字）
4. 配置你的 onebot 机器人
